README.md

```diff
@@ -40,6 +40,28 @@ jobs:
           browser: chrome
 ```
 
+### Env
+
+Specify the env argument with `env` parameter
+
+```yml
+name: Cypress tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run with env
+        uses: cypress-io/github-action@v1
+        with:
+          env: host=api.dev.local,port=4222
+```
+
+For more information, visit [the Cypress command-line docs](https://docs.cypress.io/guides/guides/command-line.html#cypress-run-env-lt-env-gt).
```

Comment:
just a tip: to keep links working we use `on.cypress.io` redirects, which are usually just title of the page (plus any hashes). So this link would be more stable as https://on.cypress.io/command-line#cypress-run-env-lt-env-gt